frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -13,6 +14,10 @@ type HeaderProperties = {
 };
 
 const Header = ({ toggleSidebar }: HeaderProperties): JSX.Element => {
+    const { email, avatarUrl } = useAppSelector(({ auth }) => ({
+        email: auth.user?.email,
+        avatarUrl: auth.user?.avatarUrl,
+    }));
```

Comment:
just select whole user and cast it to UserDto with `as` so you don't need to write all of these optional chainings etc. You sure that user will be present here, TS doesn't